samples/ChangeFeedMigrationTool/Program.cs

```diff
@@ -0,0 +1,144 @@
+// <copyright file="Program.cs" company="Microsoft">
+// Microsoft (R)  Azure SDK 
+// Software Development Kit 
+//  
+// Copyright (c) Microsoft Corporation. All rights reserved.   
+// 
+// THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,  
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES  
+// OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.  
+// </copyright>
+//---------------------------------------------------------------------------------
+
+namespace ChangeFeedMigrationSample
+{
+    using Microsoft.Azure.Documents.ChangeFeedProcessor;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.Client;
+    using System;
+    using System.Threading.Tasks;
+
+    /// ------------------------------------------------------------------------------------------------
+    /// This sample demonstrates using change processor library to read changes from source collection 
+    /// to destination collection 
+    /// ------------------------------------------------------------------------------------------------
+
+
+    class Program
+    {
+        // Modify to connect to your own subscription and database for monitored collection 
+        private string monitoredUri = "https://URI";
+        private string monitoredSecretKey = "authKey";
+        private string monitoredDbName = "monitoredDatabaseId";
+        private string monitoredCollectionName = "monitoredCollId";
+
+        // Modify to connect to your own subscription and database for lease collection 
+        // optional: setting to store lease collection on different account
+        // set lease Uri, SecretKey and DBName to same as monitored if both collections 
+        private string leaseUri = "https://URI";
+        private string leaseSecretKey = "authKey";
+        private string leaseDbName = "leaseDatabaseId";
+        private string leaseCollectionName = "leaseCollId";
+
+        // destination collection for data movement in this sample  
+        // could be same or different account 
+        private string destUri = "https://URI";
+        private string destSecretKey = "authKey";
+        private string destDbName = "leaseDatabaseId";
+        private string destCollectionName = "leaseCollId";
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Change Feed Migration Sample");
+            Program newApp = new Program();
+            newApp.RunChangeFeedProcessorAsync();
+            Console.WriteLine("Main Running... Press enter to stop.");
+            Console.ReadLine();
+        }
+
+        async void RunChangeFeedProcessorAsync()
+        {
+            // connect monitored client 
+            DocumentClient monitoredClient = new DocumentClient(new Uri(monitoredUri), this.monitoredSecretKey);
+            await monitoredClient.CreateDatabaseIfNotExistsAsync(new Database { Id = this.monitoredDbName });
```

Comment:
creating database and collection is redundant as we do that 3 times by copy-paste code. This logic should be in separate method which we call for monitored, lease and target database/collection.